src/App.vue

```diff
@@ -50,6 +53,14 @@ export default class App extends Vue {
         Axios.defaults.headers.post['Accept-Language'] = this.currentLanguage;
     }
 
+    private get maintenanceActive(): boolean {
```

Comment:
@gonzalo-plaza maybe a more accurate name would be "maintenancePageStatus", this way you are saying that this is the actual state